src/interpolations.jl

```diff
@@ -1973,4 +1966,36 @@ adjust_dofs_during_distribution(::Nedelec{RefTetrahedron, 1}) = false
 function get_direction(::Nedelec{RefTetrahedron, 1}, j, cell)
     edge = edges(cell)[j]
     return ifelse(edge[2] > edge[1], 1, -1)
-end
\ No newline at end of file
+end
+
+
+# RefHexahedron, 1st order Lagrange
+# https://defelement.org/elements/examples/hexahedron-nedelec1-lagrange-1.html
+function reference_shape_value(ip::Nedelec{RefHexahedron, 1}, ξ::Vec{3}, i::Int)
+    x, y, z = ξ
+    nil = zero(x)
```

Comment:
Two alternative ways are, but they are all equivalent. 
```suggestion
function reference_shape_value(ip::Nedelec{RefHexahedron, 1}, ξ::Vec{3, T}, i::Int) where {T}
    x, y, z = ξ
    nil = zero(T)
```

```suggestion
function reference_shape_value(ip::Nedelec{RefHexahedron, 1}, ξ::Vec{3}, i::Int)
    x, y, z = ξ
    nil = zero(eltype(ξ))
```